win_tornado_terminals/logic/term_manager.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+
+"""Term manager."""
+
+from __future__ import unicode_literals
+
+import os
+import time
+import hashlib
+import tornado.web
+import tornado.gen
+import tornado.ioloop
+
+WINDOWS = os.name == 'nt'
+
+if WINDOWS:
+    import winpty as pty
+else:
+    import pexpect as pty
```

Comment:
Up to you how best to tackle this. If it was me, I'd probably make it work as a Windows-only module for now, and then investigate what it would take to integrate with terminado once that's working.